source/dub/commandline.d

```diff
@@ -97,6 +97,16 @@ int runDubCommandLine(string[] args)
 		environment["TEMP"] = environment["TEMP"].replace("/", "\\");
 	}
 
+	// special stdin syntax
+	if (args.length >= 2 && args[1] == "-")
+	{
+		import dub.internal.utils: getTempFile;
+
+		auto path = getTempFile("app", ".d");
+		std.file.write(path.toNativeString(), stdin.byLineCopy.join);
+		args = args[0]~[path.toNativeString()]~args[2..$];
```

Comment:
No need for [] here.
Also mind the style of the existing file (and [DStyle](https://dlang.org/dstyle.html)): in other words spaces around `~`.

Reply:
fixed

Reply:
Square brackets are needed.
Args without []: ["C:\\Users\\home\\git\\dub\\bin\\dub.exeC:\\Users\home\\Desktop\\abc.d"]
Args with []: ["C:\\Users\home\\git\\dub\\bin\\dub.exe", "C:\\Users\home\\Desktop\\abc.d"]